extensions/amp-slides/0.1/amp-slides.js

```diff
@@ -14,208 +14,12 @@
  * limitations under the License.
  */
 
-import {Animation} from '../../../src/animation';
-import {isLayoutSizeDefined} from '../../../src/layout';
-import * as st from '../../../src/style';
-import * as tr from '../../../src/transition';
+import {AmpSlideScroll} from '../../amp-carousel/0.1/slidescroll';
```

Comment:
Are there existing users of `amp-slides`? In that case I think we should bump major version to 1.0